code/datums/flock/flock.dm

```diff
@@ -56,21 +56,20 @@
 
 /datum/flock/ui_act(action, list/params, datum/tgui/ui)
 	var/mob/user = ui.user;
-	if (!istype(user, /mob/living/intangible/flock/flockmind)) //no humans allowed
+	if (!istype(user, /mob/living/intangible/flock/flockmind))
 		return
 	switch(action)
 		if("jump_to")
 			var/atom/movable/origin = locate(params["origin"])
 			if(origin)
 				var/turf/T = get_turf(origin)
 				if(T.z != Z_LEVEL_STATION)
-					// make sure they're not trying to spoof data and jump into a z-level they ought not to go
 					boutput(user, "<span class='alert'>They seem to be beyond your capacity to reach.</span>")
 				else
 					user.set_loc(T)
 		if("rally")
 			var/mob/living/critter/flock/C = locate(params["origin"])
-			if(C?.flock == src) // no ordering other flocks' drones around
```

Comment:
Idk seems already explained by what's checked for, maybe more of a sanity check message would be better?

Reply:
So a lot of these either clarify intent or explain in plain English what the code is doing. To my mind, that's what comments are for - helping someone entirely unfamiliar with the language/codebase follow what's going on. I'm not going to respond to all of these because lazy, but with that criteria use your best judgement.